src/Controller/IndexController.php

```diff
@@ -137,8 +142,10 @@ public function index(Request $request)
         }
 
         if (!$tab) {
-            $this->templateParams(array('no_tab' => 1));
-
+            $this->templateParams(array(
+                'no_tab' => 1,
+                'tab'    => $tab,
```

Comment:
cette ligne ne me semble pas nécessaire, $tab est fourni en ligne 157

Reply:
Est-ce que no_tab est nécessaire ? sachant qu'on fourni $tab qui peut porter la valeur 0.
Moins de variables transmises à twig = pour de simplicité et de clarté.

On pourrait d'ailleurs simplifier le code en déplaçant le bloc `$this->templateParams(array(` de la ligne 91 après ces contrôles et y intégrer tab et no_tab (s'il reste nécessaire)